src/vm/dllimportcallback.h

```diff
@@ -211,14 +211,14 @@ class UMThunkMarshInfo
                                             // On non-x86, the managed entrypoint for no-delegate no-marshal signatures
     UINT32            m_cbActualArgSize;    // caches m_pSig.SizeOfFrameArgumentArray()
 #if defined(_TARGET_X86_)
+#if defined(FEATURE_STUBS_AS_IL)
     UINT16            m_cbRetPop;           // stack bytes popped by callee (for UpdateRegDisplay)
```

Comment:
This one should be out of the FEATURE_STUBS_AS_IL ifdef, only the offsets should be in. The m_cbRetPop is used for x86 on Windows too where the FEATURE_STUBS_AS_IL  is not defined.